sql-statements/sql-statement-explain-analyze.md

```
@@ -339,5 +339,5 @@ after key/value request is processed:
 
 -   [クエリ実行計画を理解する](/explain-overview.md)
 -   [EXPLAIN](/sql-statements/sql-statement-explain.md)
--   [ANALYZE](/sql-statements/sql-statement-analyze-table.md)
+-   [ANALYZE TABLE](/sql-statements/sql-statement-analyze-table.md)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)